src/Components/CommentForm.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react"
+import { postComment } from "./Api"
+
+
+const CommentForm = ({ setComments, article_id, updateCommentCount }) => {
+
+    const [commentInput, setCommentInput] = useState('')
+    const [usernameInput, setUsernameInput] = useState('')
+    const [submissionFeedback, setSubmissionFeedback] = useState('')
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        setSubmissionFeedback('Posting comment...')
+        postComment(article_id, usernameInput, commentInput)
+        .then((newComment) => {
+            setCommentInput('')
+            setUsernameInput('')
+            setSubmissionFeedback('Comment was sucessfully posted !')
+            setComments((prevComments) => [newComment, ...prevComments])
+            updateCommentCount()
+        })
+        .catch(() => {
+            setCommentInput('')
+            setUsernameInput('')
+            setSubmissionFeedback('Sorry, failed to post comment. Please try again.')
+        })
+    }
+
+   
+  const handleUsernameChange = (e) => {
+    setUsernameInput(e.target.value);
+  };
+
+  const handleCommentChange = (e) => {
+    setCommentInput(e.target.value);
+  };
+
+    return (
+    <>
+        <h3>Add a Comment</h3>
+        <form onSubmit={handleSubmit}>
+            <input
+              type='text' 
+              onChange={handleUsernameChange} 
+              placeholder="Type in username..." 
+              value={usernameInput} 
+              required>
+            </input>
```

Comment:
I would recommend hard coding this value instead of providing an input field for it, until you add some logic/user switching logic